src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/template.json

```diff
@@ -78,6 +78,24 @@
           "exclude": [
             "Extensions/AzureAdB2C*.cs"
           ]
+        },
+        {
```

Comment:
There appears to be a conflict between these new modifiers and this existing modifier:
```
       {
          "condition": "(!IndividualLocalAuth)",
          "exclude": [
            "Data/**",
            "Areas/**"
          ]
        },
```
If a user chooses auth = IndividualB2C, then that makes IndividualLocalAuth = false, causing the above condition to be true, which excludes the Data/ directory. 

These new modifiers, conditioned on UseLocalDB, want to include some of the content under the Data/ directory, but it's been excluded.

The existing modifier which excludes the Data/ dir might need to be revisited with this in mind.

Reply:
Looking at the rest of this review, I noticed that for StarterWeb CSharp, the modifier to exclude Data/ & Areas/ is conditioned on "(!IndividualAuth)", which might be the desired change here.

Reply:
Well it shouldn't matter as the rename won't do anything if the top level Data was all excluded right?  But seems harmless to add an additional IndividualLocalAuth check for these new conditions...

Reply:
Correct, the rename won't do anything if the Data folder is already excluded. 

But it seems to be the case that with auth = IndividualB2C, we'd want to include the appropriate migration files - based on the UseLocalDB value. If this is incorrect, then it's probably fine as is.

Reply:
IndividualB2C = true means IndividualLocalAuth = false...  so the migrations should and are excluded.  I'm happy to add the IndividualLocalAuth check if you think that makes it clearer in the long run...  its not exactly obvious how all this stuff plays together at all, so even redundant checks have some value for clarity in the future

Reply:
I don't feel too strongly either way about adding the additional check.

The reason brought this up originally is because the description on the UseLocalDB parameter indicates that it is applicable for both Individual & IndividualB2C auth types. With that in mind, I assumed that migrations would be desired for B2C.

Reply:
These are IndividualAuth specific migrations so that's why